src/main/kotlin/week1/step3/RacingCar.kt

```diff
@@ -0,0 +1,16 @@
+package week1.step3
+
+fun main() {
+    val (carCount, tryCount) = InputView.getCarAndTryCount()
+    val racingCar = Array(carCount) { Car() }
+
+    showContent(tryCount, racingCar)
+}
+
+private fun showContent(tryCount: Int, racingCar: Array<Car>) {
+    for (count in 0 until tryCount) {
```

Comment:
`repeat()`를 찾아보고 그 기능을 사용해 보면 어떨까요?

```suggestion
    repeat(tryCount) {
```

Reply:
찾고 싶었던 함수였습니다.
감사합니다.:+1:
[3bd873ed92294d23f7ef550b0b9e10a3f0c73483]